bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/filters/WhileMatchFilterAdapter.java

```diff
@@ -75,6 +75,9 @@ public WhileMatchFilterAdapter(FilterAdapter subFilterAdapter) {
    */
   @Override
   public RowFilter adapt(FilterAdapterContext context, WhileMatchFilter filter) throws IOException {
+    checkArgument(
+        context.getNumberOfWhileMatchFilters() == 0,
```

Comment:
Maybe make it a get/set and check that get returns null? its kind of weird to allow a list but enforce a maximum of 1.